src/cryptonote_core/service_node_quorum_cop.h

```diff
@@ -80,6 +80,7 @@ namespace service_nodes
     bool voted_in_checkpoints     = true;
     bool storage_server_reachable = true;
 
+    char const *why(bool testing_myself) const;
```

Comment:
It looks like this is always called with `testing_myself` set to `true`

Reply:
Removed